xbmc/cores/RetroPlayer/rendering/RPRenderManager.cpp

```diff
@@ -74,6 +80,13 @@ void CRPRenderManager::Deinitialize()
     buffer->Release();
   m_pendingBuffers.clear();
 
+  for (auto it : m_savestateBuffers)
```

Comment:
```suggestion
  for (auto& it : m_savestateBuffers)
```

Reply:
you could also possibly do something cool like:

```suggestion
  for (auto& [saveStatePath, renderBuffers] : m_savestateBuffers)
```